### ISSUE TITLE
Set up online/offline status check

### ISSUE THREAD

**Issue by Banjerr** [Banjerr/electrello#2]
Set up check for online/offline status so that the when offline the changes are stored locally and once online the changes are "pushed" up to Trello in the background. When online the edits will go straight to Trello.

http://electron.atom.io/docs/tutorial/online-offline-events/

**Comment by austinaryain** [comment]
I would love to take this one if its still needed?

I saw on your Trello board that #3  may be resolved?  I wouldn't mind helping with that one too in the future.

**Comment by Banjerr** [comment]
yep, still needed! #3 was improved, but i'm certainly open to alternatives if you know of any better!

**Comment by GrosSacASac** [comment]
why are you not merging existing PRs ?
